lib/sqlalchemy/dialects/mysql/mariadbconnector.py

```diff
@@ -201,19 +223,21 @@ def create_connect_args(self, url):
             except (AttributeError, ImportError):
                 self.supports_sane_rowcount = False
             opts["client_flag"] = client_flag
-        return [[], opts]
+        return [], opts
 
-    def _extract_error_code(self, exception):
+    def _extract_error_code(self, exception: BaseException) -> int:
         try:
-            rc = exception.errno
+            rc: int = exception.errno  # type: ignore
         except:
             rc = -1
         return rc
 
-    def _detect_charset(self, connection):
+    def _detect_charset(self, connection: "Connection") -> str:
         return "utf8mb4"
 
-    def get_isolation_level_values(self, dbapi_connection):
+    def get_isolation_level_values(
+        self, dbapi_conn: "DBAPIConnection"
+    ) -> tuple["IsolationLevel", ...]:
```

Comment:
```suggestion
        self, dbapi_conn: DBAPIConnection) -> Sequence[IsolationLevel];
```
(quotes removed, return type consistent with parent)